deep_quoridor/src/quoridor_env.py

```diff
@@ -297,45 +260,7 @@ def _get_info(self):
         return {}
 
     def render(self):
-        board_str = ""
-
-        for row in range(self.board_size):
-            # Render board row with player positions and vertical walls
-            for col in range(self.board_size):
-                if self.positions["player_0"] == (row, col):
-                    board_str += " P0 "
-                elif self.positions["player_1"] == (row, col):
-                    board_str += " P1 "
-                else:
-                    board_str += " .  "
-
-                # Render vertical walls (|)
-                if col < self.wall_size:
-                    if row < self.wall_size and self.walls[row, col, 0]:
-                        board_str += "|"
-                    elif row > 0 and self.walls[row - 1, col, 0]:  # Continuation of a vertical wall
-                        board_str += "|"
-                    else:
-                        board_str += " "
-                else:
-                    board_str += " "
-
-            board_str += "\n"
-
-            # Render horizontal walls (───)
-            if row < self.wall_size:
-                for col in range(self.board_size):
-                    if col < self.wall_size and self.walls[row, col, 1]:
-                        board_str += "────+"  # Wall segment
-                    elif col > 0 and self.walls[row, col - 1, 1]:
-                        board_str += "──── "  # Continuation of a wall from the left
-                    elif col < self.wall_size and self.walls[row, col, 0]:
-                        board_str += "    +"  # Vertical wall going through this intersection
-                    else:
-                        board_str += "     "
-                board_str += "\n"
-
-        print(board_str)
+        print(self._game.render_to_string())
```

Comment:
I don't see this function defined

Reply:
Luckily the tests fail long before this point so i never notice :-P